introducing-polyglotweekly.com.md

```diff
@@ -0,0 +1,193 @@
+* I wanted a platform that rendered source-code in a beautiful way,
+  other blogging platforms I've used have been so so at this.
+* I'd love to read articles from a variety programming communities.
+  We can all learn from each other's methodologies, and it's important to discourage  inter-community shade throwing.
+* I love working with an editor. As frustrating as it can sometimes be, it helps
+  ensure that I write quality articles. I wanted to create a platform that
+  had an editorial process.
+
+## How Polyglot Weekly Works
+
+### Generating an Article
+
+Anyone can submit a pull request for an article to Polyglot Weekly, simply run:
+
+```shell
+git clone https://github.com/polyglotweekly/polyglotweekly.com.git
+cd polyglotweekly.com
+nvm use 0.10 # sorry, no 0.12.x or iojs yet.
+npm install
+npm run generate
+```
+
+Keep the following contribution guidelines in mind:
+
+* Articles should be on the topic of programming: articles advertising your start-up,
+  discussing Bay-Area culture, or pointing out your favorite dog grooming techniques will be politely declined.
+* keep contributions positive! your article should not be a rant about why
+  Ruby is better than Go. Feel free to criticize aspects of your language
+  of expertise, but do so in a constructive manner.
+* Provide concrete examples in code. Polyglot Weekly uses a [great syntax-highlighter](https://github.com/atom/highlights) extracted from the Atom Text Editor, take advantage of it.
+
+### The Editorial Process
+
+Articles are submitted to the [polyglotweekly.com repository](https://github.com/polyglotweekly/polyglotweekly.com) as a pull request. This allows a conversation to take place with the community surrounding the article. This discussion may include:
+
+* grammatical fixes and other editorial nits.
+* suggestions about structure, e.g., adding a section to an article that might
+  pull the whole piece together.
+* as with the tone of articles, the goal will be to keep the editorial process positive.
+
+Once the conversation surrounding an article is complete, it will be merged
+on to master and become visible on [www.polyglotweekly.com](http://www.polyglotweekly.com/).
+
+## The Tech Behind Polyglot Weekly
+
+The Polyglot Weekly platform is built with some really cool technology,
+let me give you a tour:
+
+### Markdown Rendering
+
+[marky-markdown](https://www.npmjs.com/package/marky-markdown) is used for rendering the articles. This is the same parser used for rendering the README files
+on www.npmjs.com. marky-markdown adheres closely to the [CommonMark Specification](http://spec.commonmark.org/), which is worth giving a read if
+you'd like to contribute an article.
+
+### Syntax Highlighting
+
+Polyglot Weekly uses the same engine as the [Atom Text Editor](https://atom.io/) for syntax-highlighting. A variety of languages are supported:
+
+**shell**
+
+```shell
```

Comment:
I had the same thought, I think I went a bit overboard because I was having fun playing with the syntax highlighter. Perhaps leave a couple concrete examples?